analytical/tests/test_tag_uservoice.py

```diff
@@ -58,7 +58,7 @@ def test_override_options(self):
         r = UserVoiceNode().render(Context(data))
         self.assertIn("""UserVoice.push(['set', {"key1": "val2"}]);""", r)
 
-    def test_auto_trigger(self):
+    def test_auto_trigger_default(self):
```

Comment:
@jcassee This is what I found (and hopefully fixed correctly) by help of `flake8`.